rocAL/rocAL/source/image_loader_sharded.cpp

```diff
@@ -22,12 +22,23 @@ THE SOFTWARE.
 
 #include "image_loader_sharded.h"
 
+#if ENABLE_HIP
+ImageLoaderSharded::ImageLoaderSharded(DeviceResourcesHip dev_resources):
+#else
 ImageLoaderSharded::ImageLoaderSharded(DeviceResources dev_resources):
+#endif
         _dev_resources(dev_resources)
 {
     _loader_idx = 0;
 }
 
+void ImageLoaderSharded::set_prefetch_queue_depth(size_t prefetch_queue_depth)
+{
+    if(prefetch_queue_depth <= 0)
```

Comment:
I think it is better to set it some min value like 3. Does the pipeline work if the prefetch_q size is set to 1?

Reply:
The default value for prefetch_queue_depth is 3 and is set in rali_api.h